.eslintrc.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+    'root': true,
+    'env': {
+        'node': true
+    },
+    'extends': [
+        'eslint:recommended',
+        'plugin:unicorn/recommended',
+    ],
+    'parser': '@typescript-eslint/parser',
+    'parserOptions': {
+        'ecmaVersion': 2017,
+        'sourceType': 'module',
+        'ecmaFeatures': {
+            'jsx': false,
+        },
+        'project': 'tsconfig.json',
+    },
+    'plugins': [
+        'unicorn',
+        'import',
+        '@typescript-eslint/tslint',
```

Comment:
According to the [documentation](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin-tslint) of [`@typescript-eslint/tslint`](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin-tslint) this runs `tslint` on top of `eslint`. That is _not_ what we want. We want to _remove_ `tslint` completely. 

Use [`@typescript-eslint`](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin) instead. It's more work than just "copy paste".